commitlint.config.ts

```diff
@@ -76,6 +76,15 @@ function isFooterNote(line: string): boolean {
         isFixesSentence(line);
 }
 
+function isName(word: string) {
```

Comment:
@realmarv this needs an assert

Reply:
@realmarv while you're at it, let's rename the function name to wordIsStartOfSentence

Reply:
> @realmarv while you're at it, let's rename the function name to wordIsStartOfSentence

I didn't get why. Because this function is making sure the first character is uppercase and the rest of the characters are lowercase and there are no non-alphabetical characters in it, which I guess is the definition of a name.